etl/steps/data/garden/eth/2023-03-15/ethnic_power_relations.meta.yml

```diff
@@ -0,0 +1,594 @@
+          - *description_ethnic_group
+          - *description_egip
+          - *description_autonomy
+        unit: "%"
+        short_unit: "%"
+        display:
+          name: Share of population with regional autonomy and in an EGIP group
+          numDecimalPlaces: 1
+        sources:
+          - *source-epr
+      cntr_relevance:
+        title: Relevance
+        description:
+          - '"R" indicates countries where ethnicity is coded as being relevant at least once in the sample period; "P" indicates countries where only a placeholder group is coded.'
+          - *description_ethnic_group
+          - *description_group_relevance
+        unit: ""
+        short_unit: ""
+        display:
+          name: Relevance
+        sources:
+          - *source-epr
+      nstar:
+        title: Index N* of ethnonationalist exclusiveness
+        description:
+         - "N*(0.5; 5);  see Cederman L.-E. and L. Girardin (2007). Beyond fractionalization: Mapping ethnicity onto nationalist insurgencies. The American Political Science Review 101(1): pp. 173-185."
+        unit: ""
+        short_unit: ""
+        display:
+          name: N*
+        sources:
+          - *source-epr
+      onset_ko_eth_flag:
+        title: Ethnic conflict onset (KO)
+        description:
+          - Binary flag indicating ethnic conflict onset / ko option.
+          - *description_ethnic_group
+          - *description_acd_conflict
+          - *description_conflict_onset
+        unit: ""
+        short_unit: ""
+        display:
+          name: Ethnic conflict onset (KO)
+          numDecimalPlaces: 0
+        sources:
+          - *source-acd2epr
+      onset_ko_noneth_flag:
+        title: Nonethnic conflict onset (KO)
+        description:
+          - Binary flag indicating nonethnic conflict onset / ko option
+          - *description_ethnic_group
+          - *description_acd_conflict
+          - *description_conflict_onset
+        unit: ""
+        short_unit: ""
+        display:
+          name: Nonethnic conflict onset (KO)
+          numDecimalPlaces: 0
+        sources:
+          - *source-acd2epr
+      onset_ko_terr_eth_flag:
+        title: Territorial ethnic conflict onset (KO)
+        description:
+          - Binary flag indicating territorial ethnic conflict onset / ko option
+          - *description_ethnic_group
+          - *description_acd_conflict
+          - *description_conflict_onset
+        unit: ""
+        short_unit: ""
+        display:
+          name: Territorial ethnic conflict onset (KO)
+          numDecimalPlaces: 0
+        sources:
+          - *source-acd2epr
+      onset_ko_gov_eth_flag:
+        title: Governmental ethnic conflict onset (KO)
+        description:
+          - Binary flag indicating governmental ethnic conflict onset / ko option
+          - *description_ethnic_group
+          - *description_acd_conflict
+          - *description_conflict_onset
+        unit: ""
```

Comment:
Do you know the reason why the jump spaces behave differently in cases like [this variable](https://owid.cloud/admin/variables/576990), @Marigold? When there's a text instead of the anchor (like in the first item of the list), there is no space between this and the second item. And when I use an HTML list (as `*description_acd_conflict`) there seems to be a double jump with `*description_conflict_onset`.

No space:
<img width="564" alt="image" src="https://user-images.githubusercontent.com/63430031/233435216-73337d9e-eeae-495d-ad4d-dabc155cb4b2.png">

Double space:
<img width="482" alt="image" src="https://user-images.githubusercontent.com/63430031/233435276-69ac2ea5-15f5-423e-8a02-6bd359aae824.png">

Reply:
If you mean the space before "Each conflict..." then that's not actually a double space, but a space and bottom margin for `<ul>` element. If you want you can eliminate the space by writing it as
```
- acd_conflict:
      description: &description_acd_conflict |
        An ACD (Armed Conflict Database) conflict for this dataset includes:
            <ul>
              <li>Intrastate conflict: A conflict between a government and a non-governmental party, with no interference from other countries.</li>
              <li>Intrastate conflict internationalized: An armed conflict between a government and a non-government party where the government side, the opposing side, or both sides, receive troop support from other governments that actively participate in the conflict.</li>
            </ul>Each conflict is assigned a unique UCDP (Uppsala Conflict Data Program) ID. ACD conflicts are merged with the Ethnic Power Relations country-level data by assigning each country-year observation one or several UCDP IDs if the ACD dataset reports an ongoing conflict in the respective country and year.
```

Reply:
Ah, you're right, thanks! And how about the first case?

Reply:
@paarriagadap sorry, can't replicate it now. I've just uploaded the [dataset to staging](https://staging.owid.cloud/admin/datasets/5921). Could you tell me what variable is problematic?

Reply:
Any variable, for instance [this one](https://staging.owid.cloud/admin/variables/576837). There is no jump between the first and the second paragraph. In this case, I have formatted the first paragraph not as an anchor but a text.

Reply:
I see. Try adding `<br>` at the end of all texts, that should help, i.e.

```
description:
- Count variable indicating the number of EGIP groups in this country.<br>
- *description_ethnic_group
- *description_egip
```

I should really get those placeholders working...

Reply:
Thanks! I did that last time with the multidimensional poverty index, but I thought I could do something else

Reply:
I will let you know, @Marigold. Thank you very much!